scarb/src/ops/new.rs

```diff
@@ -20,6 +21,7 @@ pub struct InitOptions {
     pub path: Utf8PathBuf,
     pub name: Option<PackageName>,
     pub vcs: VersionControl,
```

Comment:
`name` and `vcs` args would be ignored?

Reply:
`name` is used and `vcs` is ignored by forge

Reply:
If we allow to use both `--no-vcs` and `--snforge` simultaneously - we should support it.

Reply:
I believe snforge is NOT touching vcs stuff; and this code is handling VCS by itself

Reply:
Ok, cool, makes sense!